schemas/commands/appeal-representation-submission.schema.json

```diff
@@ -0,0 +1,169 @@
+                    "required": [],
+                    "properties": {
+                        "firstName": {
+                            "type": "string",
+                            "examples": ["Alan"]
+                        },
+                        "lastName": {
+                            "type": "string",
+                            "examples": ["Turing"]
+                        },
+                        "emailAddress": {
+                            "type": ["string", "null"],
+                            "examples": ["john.doe@example.com"]
+                        },
+                        "address": {
+                            "title": "Address",
+                            "type": "object",
+                            "required": ["addressLine1", "town", "postcode"],
+                            "properties": {
+                                "addressLine1": {
+                                    "type": "string",
+                                    "examples": ["National Traffic Operations Centre"]
+                                },
+                                "addressLine2": {
+                                    "type": "string",
+                                    "examples": ["3 Ridgeway, Quinton Business Park"]
+                                },
+                                "town": {
+                                    "type": "string",
+                                    "examples": ["Birmingham"]
+                                },
+                                "county": {
+                                    "type": ["string", "null"],
+                                    "examples": ["Birmingham"]
+                                },
+                                "postcode": {
+                                    "type": "string",
+                                    "examples": ["B32 1AF"]
+                                }
+                            }
+                        }
+                    }
+                },
+                "representationType": {
+                    "type": ["string", "null"],
+                    "enum": ["statement", "comment", "final_comment", "proofs_evidence", null],
+                    "description": "The type of representation"
+                },
+                "representation": {
+                    "type": ["string", "null"],
+                    "examples": ["I have concerns"]
+                },
+                "submittedDate": {
+                    "type": ["string"],
+                    "format": "date-time",
+                    "examples": ["2023-07-27T20:30:00.000Z"],
+                    "description": "The date the statement was submitted by the user"
+                }
+            },
+            "oneOf": [
+                { "required": ["serviceUserId"] },
+                { "required": ["lpaCode"] },
+                { "required": ["newUser"] }
```

Comment:
Existing appellant/agent
LPA
newUser is for IP comments where any member of public can comment and supply limited personal info, could have a better name

Reply:
may need to be just nullable service user if we remove `lpaCode`